common/crypto/ledger/ledger.go

```diff
@@ -0,0 +1,21 @@
+// +build cgo,ledger
+
+package ledger
+
+import (
+	ledger "github.com/zondax/ledger-cosmos-go"
```

Comment:
Why not just import `github.com/binance-chain/ledger-cosmos-go`? Seems the source override is not needed.

Reply:
Good idea.